api-spec/extensions/query/README.md

```diff
@@ -44,3 +44,28 @@ Find scenes with cloud cover between 0 and 10%:
   }
 }
 ```
+
+## Design Note
```

Comment:
I don't think this design note is necessary, it's just providing a justification for why this wasn't done some other way, and don't think we should be in the habit of feeling like we have to do so.  I've recently removed other similar design notes from other parts of the spec.

Reply:
Agreed.

Reply:
removed.